features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2762,4 +2762,10 @@
         	<Cookie name="{{cookie}}"/>
         	{% endfor %}
     </CookiesToInvalidate>
+
+    <!-- Claim to store localization code  -->
+    <!--    By setting true localization code will be stored inside the http://wso2.org/claims/locality-->
+    <!--    By setting false localization code will be stored inside the http://wso2.org/claims/local-->
+    <isLegacyLocalityClaimUri>true</isLegacyLocalityClaimUri>
```

Comment:
let's make the value configurable from deployment.toml.
`<useLegacyLocalizationClaim>{{use_legacy_localization_claim}}</useLegacyLocalizationClaim>`

Reply:
let's change this like follows.

```
{% if use_legacy_localization_claim is defined %}
        <useLegacyLocalizationClaim>{{use_legacy_localization_claim}}</useLegacyLocalizationClaim>
        {% else %}
        <useLegacyLocalizationClaim>false</useLegacyLocalizationClaim>
        {% endif %}
```

Reply:
Fixed in commit 40f50964fb888adb7c5f78d42bd804eb164c13e8